From patchwork Fri Jul  3 17:13:35 2020
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
Subject: [EXTERNAL,
 tisdk-setup-scripts] setup-targetfs-nfs: Update the rootfs to default
Date: Fri, 03 Jul 2020 23:13:35 -0000
From: Praveen Rao <prao@ti.com>
X-Patchwork-Id: 55971
Message-Id: <20200703171335.8138-1-prao@ti.com>
To: <meta-arago@arago-project.org>, <denys@ti.com>, <j-stiffler@ti.com>,
 <nikhil.nd@ti.com>
Cc: Praveen Rao <prao@ti.com>

From: Praveen Rao <prao@ti.com>

The filesystem has change to tisdk-default-image.tar.xz

Signed-off-by: Praveen Rao <prao@ti.com>

---
setup-targetfs-nfs.sh | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/setup-targetfs-nfs.sh b/setup-targetfs-nfs.sh
index 8f7fbb1..c3062e9 100644
--- a/setup-targetfs-nfs.sh
+++ b/setup-targetfs-nfs.sh
@@ -59,19 +59,19 @@ extract_fs() {
     fstar=""
     while [ -z "$fstar" ]
     do
-        numfs=`ls $cwd/../filesystem | grep "tisdk.*rootfs" | grep 'tar.xz' | grep -n '' | grep '2:' | awk {'print $1'}`
+        numfs=`ls $cwd/../filesystem | grep "tisdk.*default" | grep 'tar.xz' | grep -n '' | grep '2:' | awk {'print $1'}`
         if [ -n "$numfs" ]
         then
             echo
             echo "Multiple filesystems found."
-            ls --sort=size $cwd/../filesystem | grep "tisdk.*rootfs" | grep 'tar.xz' | grep -n '' | awk {'print "       " , $1'}
+            ls --sort=size $cwd/../filesystem | grep "tisdk.*default" | grep 'tar.xz' | grep -n '' | awk {'print "       " , $1'}
             echo
             read -p "Enter Number of rootfs Tarball: [1] " fsnum
             [ -n "$fsnum" ] || fsnum=1
             echo
-            fstar=`ls --sort=size $cwd/../filesystem | grep "tisdk.*rootfs" | grep 'tar.xz' | grep -n '' | grep "^$fsnum:" | cut -c3- | awk {'print$1'}`
+            fstar=`ls --sort=size $cwd/../filesystem | grep "tisdk.*default" | grep 'tar.xz' | grep -n '' | grep "^$fsnum:" | cut -c3- | awk {'print$1'}`
         else
-            fstar=`ls  $cwd/../filesystem | grep "tisdk.*rootfs" | grep 'tar.xz' | awk {'print $1'}`
+            fstar=`ls  $cwd/../filesystem | grep "tisdk.*default" | grep 'tar.xz' | awk {'print $1'}`
         fi
         if [ -z "$fstar" ]
         then
